### DISCUSSION TITLE
Smart coupon - Especially Credit system

### DISCUSSION THREAD

**Discussion by Gardens18** [inspireui/support#5134] [New Feature/Ideas Requests]
https://woocommerce.com/products/smart-coupons/?quid=1a5de4f9c8fa28b9a93cb7b54f722070

We need this feature.

**Comment by meek2100** [comment]
+1 on adding full support for the Smart Coupon plugin. 

Also be aware that using this plugin with an usage restricted coupons today will cause issue with FluxStore Pro.

**Comment by ameenbadri7** [comment]
Smart coupon also the built-in Credit system are amazing features to impelement on the fluxstore pro.

**Comment by Uhtredtb** [comment]
Please make it support the app and multi vendor @minhcasi

**Comment by Uhtredtb** [comment]
Hi, please support this app

**Comment by quangbahoa** [comment]
Any update for this feature @minhcasi  ? Thanks
